src/MSBuild/XMake.cs

```diff
@@ -2912,18 +2914,43 @@ private static CommandLineSwitches CombineSwitchesRespectingPriority(CommandLine
             return commandLineSwitches;
         }
 
-        private static bool CheckAndGatherProjectAutoResponseFile(CommandLineSwitches switchesFromAutoResponseFile, CommandLineSwitches commandLineSwitches, bool recursing, string commandLine, out string projectFile)
+        private static string GetProjectDirectory(string[] projectSwitchParameters)
         {
-            bool found = false;
+            string projectDirectory = ".";
+            ErrorUtilities.VerifyThrow(projectSwitchParameters.Length <= 1, "It should not be possible to specify more than 1 project at a time.");
```

Comment:
It isn't now, so this assert is good, but I'd change the comment since some users have asked for this (though I can't find an issue).
```suggestion
            ErrorUtilities.VerifyThrow(projectSwitchParameters.Length <= 1, "Expect exactly one project at a time.");
```

Reply:
Updated the text, thank you!